test/core/storage/trie/polkadot_trie/polkadot_trie_test.cpp

```diff
@@ -332,3 +332,19 @@ TEST_F(TrieTest, EmptyTrie) {
   EXPECT_OUTCOME_TRUE_1(trie->put({0}, "asdasd"_buf));
   ASSERT_FALSE(trie->empty());
 }
+
+TEST_F(TrieTest, GetPath) {
```

Comment:
better make parameterized test

Reply:
Agree with you, but I'd prefer to create a separate task for it, as this test happened to be not that easy to write.